src/api/protectedApiClient.ts

```diff
@@ -6,13 +6,46 @@ export interface ProtectedApiClient {
     newPassword: string;
   }) => Promise<void>;
   readonly deleteUser: (request: { password: string }) => Promise<void>;
+  readonly postOnboardingForm: (
+    request: OnboardingRequestData,
+  ) => Promise<OnboardingRequestData>;
+  readonly getOnboardingData: () => Promise<OnboardingResponseData[]>;
 }
 
 export enum ProtectedApiClientRoutes {
   CHANGE_PASSWORD = '/api/v1/protected/user/change_password',
   DELETE_USER = '/api/v1/protected/user/',
 }
 
+export interface OnboardingRequestData {
```

Comment:
Would prefer if we named this after than "onboarding"
```suggestion
export interface CreatePostRequest{
```